packages/integration_test/example/tizen/Runner.csproj

```
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Tizen.NET.Sdk/1.1.5">
+﻿<Project Sdk="Tizen.NET.Sdk/1.1.6">
```

```suggestion
﻿<Project Sdk="Tizen.NET.Sdk/1.1.7">
```
`1.1.7` is the latest verison.

---

Okay